heaps/min_heap.py

```diff
@@ -57,19 +74,54 @@ def heap_up(self, index):
             property is reestablished.
             
             This could be **very** helpful for the add method.
-            Time complexity: ?
-            Space complexity: ?
+            Time complexity: log n
+            Space complexity: 1
```

Comment:
✨ Nice, however because of the recursive call stack, space complexity is O(log n) here